plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/files/XmlReload.java

```diff
@@ -0,0 +1,34 @@
+package org.hotswap.agent.plugin.spring.files;
+
+import org.hotswap.agent.logging.AgentLogger;
+import org.springframework.beans.factory.support.DefaultListableBeanFactory;
+
+import java.net.URL;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class XmlReload {
```

Comment:
I think we should remove this class. Call XmlBeanDefinitionScannerAgent.reloadXmls() directly from SpringReload.